src/theory/theory_inference_manager.cpp

```diff
@@ -388,6 +394,39 @@ bool TheoryInferenceManager::processInternalFact(TNode atom,
   Trace("infer-manager") << "TheoryInferenceManager::assertInternalFact: "
                          << (pol ? Node(atom) : atom.notNode()) << " from "
                          << expn << std::endl;
+#ifdef CVC5_ASSERTIONS
```

Comment:
Could we use `Configuration::isAssertionBuild()` instead?

Reply:
I've updated this.  I assume this will be 100% compiled away in non-assertion builds?  Asking since this is performance critical code.